### ISSUE TITLE
espnet1 vivos recipe RNNT training error

### ISSUE THREAD

**Issue by sairitwik27** [espnet/espnet#3418]
Hi,
I am trying to run the vivos asr1 recipe in espnet1 with the following train.yaml config file(it's the default file given):

# minibatch related
batch-size: 30
maxlen-in: 800
maxlen-out: 150

# optimization related
criterion: loss
early-stop-criterion: "validation/main/loss"
sortagrad: 0
opt: noam
transformer-lr: 10.0
transformer-warmup-steps: 8000
epochs: 30
patience: 0
accum-grad: 4
grad-clip: 5.0

# network architecture
## general
custom-enc-positional-encoding-type: rel_pos
custom-enc-self-attn-type: rel_self_attn
## encoder related
etype: custom
custom-enc-input-layer: vgg2l
enc-block-arch:
        - type: conformer
          d_hidden: 320
          d_ff: 320
          heads: 4
          macaron_style: True
          use_conv_mod: True
          conv_mod_kernel: 15
          dropout-rate: 0.4
          pos-dropout-rate: 0.4
          att-dropout-rate: 0.4
enc-block-repeat: 8
## decoder related
dtype: lstm
dlayers: 1
dec-embed-dim: 512
dunits: 512
dropout-rate-decoder: 0.2
dropout-rate-embed-decoder: 0.2
## joint network related
joint-dim: 128

# transducer related
model-module: "espnet.nets.pytorch_backend.e2e_asr_transducer:E2E"

I encounter the following error. I have tried other transducer config files in the directory as well and I get the same error.

asr_train.py: error: argument --etype: invalid choice: 'custom' (choose from 'transformer', 'lstm', 'blstm', 'lstmp', 'blstmp', 'vgglstmp', 'vggblstmp', 'vgglstm', 'vggblstm', 'gru', 'bgru', 'grup', 'bgrup', 'vgggrup', 'vggbgrup', 'vgggru', 'vggbgru')

**Comment by b-flo** [comment]
Hi,

Sorry, most configurations are broken and we didn't keep them updated because there was some changes on the way which would break them again. All configurations will be updated shortly after #3217 is merged and new ones will be added.

Having said that, the configuration you linked should work with current master. 

> asr_train.py: error: argument --etype: invalid choice: 'custom' (choose from 'transformer', 'lstm', 'blstm', 'lstmp', 'blstmp', 'vgglstmp', 'vggblstmp', 'vgglstm', 'vggblstm', 'gru', 'bgru', 'grup', 'bgrup', 'vgggrup', 'vggbgrup', 'vgggru', 'vggbgru')

`etype: custom` was introduced in #2898 (27th january). Is your repo up-to-date? If it is, could you please share the complete log?

**Comment by sairitwik27** [comment]
Ok,sorry just checked,looks like my repo is not up to date. Thanks a lot for the help.
